src/assets/slides/budapest-ml-2025_files/libs/quarto-diagram/mermaid-postprocess-shim.js

```
@@ -0,0 +1,46 @@
+};
+
+// deno-lint-ignore no-window-prefix
+window.addEventListener(
+  "load",
+  function () {
+    for (const svgEl of Array.from(
+      document.querySelectorAll("div.cell-output-display svg")
+    ).filter(
+      (el) =>
+        el.querySelector("desc").id &&
+        el.querySelector("desc").id.startsWith("chart-desc-mermaid")
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)